### ISSUE TITLE
view navigation

### ISSUE THREAD

**Issue by mozg4D** [SoftFever/OrcaSlicer#5026]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Unfortunately view navigation is a pain in orca slicer, especially for large objects It would be nice to implement the following:
1. Orbit: about mouse cursor in world space (I point with cursor on specific location on object and orbit about that point)
2. Pan: should depend on distance from cursor in world space to camera (should feel like I click and drag the view like my cursor is linked to an object)
3. Cam FOV should not depend on size of build volume / should be static 55deg (now FOV is weird for large build volumes)

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

Same as view navigation in Cinema4D

**Comment by Ataraxia-Mechanica** [comment]
Being able to switch between perspective/isometric mode would also be cool. And hotkey customization would be useful as well.

**Comment by icariantk** [comment]
+1 on the Hotkey customization, or a way to link USB Joystick/ Input device to a navigation function, i could use the Force Feedback joystick from 1000 years ago to be useful again.

**Comment by haski369** [comment]
Also, Being zoomed in on object to check some settings and reslice after changing something - Camera auto centers,- it shouldn't because I'm looking at the exact spot I need, and have to get myself back to the needed part.

In other words - Save camera position in preview even after reslicing after changing some settings (Optional button in the settings will do that too.)

**Comment by CCS86** [comment]
OS could really benefit from view improvements like these.

**Comment by Stormwing01** [comment]
Yes please, the current viewport navigation gives me motion sickness with how you are forced to pinwheel around the center plate instead of where I'm actually looking.

**Comment by icariantk** [comment]
Bump for unstaleing

**Comment by RandomGgames** [comment]
Bump
